discovery-infra/test_infra/helper_classes/nodes.py

```diff
@@ -101,10 +101,11 @@ def get_bootstrap_node(self, cluster) -> Node:
 
     def create_nodes_cluster_hosts_mapping(self, cluster):
         node_mapping_dict = {}
-        for cluster_host_object in cluster.get_nodes():
+        for cluster_host_object in cluster.get_hosts():
             inventory = json.loads(cluster_host_object["inventory"])
```

Comment:
i think we can move logic getting hostname to function

Reply:
or i think we even have it

Reply:
changed